docs/workflows/templates.md

```diff
@@ -0,0 +1,61 @@
+## Content Templates
+
+## Background
+
+Content Templates allow users to set a baseline of repository snapshots that help them control patching in predictable ways. In the future,
+users will be able to build images with a Content Template.
+
+### What is a Content Template?
+
+Content Templates are entities that can include snapshots from different Red Hat and custom repositories. 
+
+The content included in Content Templates is served at different distribution paths in pulp for Red Hat and custom since they are in different domains.
```

Comment:
```suggestion
The content included in Content Templates is served at different distribution paths in pulp for Red Hat and custom repositories since they are in different domains.
```